### PR TITLE
baking the new images into the vhd.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -268,24 +268,23 @@ if [[ ${installSGX} == "True" ]]; then
 fi
 
 TUNNELFRONT_VERSIONS="
-v1.9.2-v3.0.11
-v1.9.2-v4.0.11
-v1.9.2-v3.0.12
-v1.9.2-v4.0.12
-v1.9.2-v3.0.13
 v1.9.2-v3.0.14
+v1.9.2-v3.0.16
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:272]: no 3.0.15?

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:272]: actually are we using 3.0.x? I see the toggle only use 4.0.x?

Comment (andyliuliming) [reply vhdbuilder/packer/install-dependencies.sh:272]: we have two versions, most of us should be using 3.0.x
one is without gateway, one is (for old underlay, but we already removed them all?, not sure).

Comment (andyliuliming) [reply vhdbuilder/packer/install-dependencies.sh:272]: for some reason, the 3.0.15 is skipped.

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:272]: https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-config?path=%2Ftoggles%2Ftunnel-image-tag.yaml

the toggle is 4.0.x, why would most of us be using 3.0.x?

**Pull request merged**